hazelcast-jet-distribution/src/root/release_notes.txt

```diff
@@ -5,6 +5,78 @@ issues at https://github.com/hazelcast/hazelcast-jet/issues/<issue number>.
+[core] Replace JetBoostrap.getInstance() with Jet.bootstrappedInstance() [#1827]
+[code-samples] Add examples for Cloud Object Storages using Hadoop [#1623]
+[documentation] Write deployment and operations guide for Jet
+[documentation] Add migration guide to new Jet versions [#1947]
+[documentation] Explain how to parametrise jobs when submitting it using jet.sh [#1794]
+[documentation] Document S3 sink behaviour and limitation for streaming jobs [#1795]
+[distribution] Make Jet distribution more dev-ops friendly [#1703]
+[distribution] Auto import modules in JDK9+ in start script [#1737]
+[distribution] Introduce ability to add Jars in ZIP files as job resources [#1681]
+[distribution] Include slf4j-logj12 in the distribution [#1718]
+[distribution] Enhance Hadoop JARs and Classpath chapter in reference manual [#1830]
+[jet-cli] Add daemon mode and rolling file logger [#1783]
+[jet-cli] Introduced an option to pick main class from JAR files [#1800]
+[jet-cli] Allow for the selection of an entry class when submitting job from the CLI [#1796]
+[integration] Move connectors and plugins to the Hazelcast Community License [#1892]
+[integration] Slim down and improve JMS sink [#1807]
+[integration] Introduce a new variant in the HDFS source which uses new API [#1636]
+[integration] Make reading of frequently used file formats convenient (CSV, JSON, Parquet, Avro) [#1849]
+[integration] Unify Source Context and Sink Writer [#1785]
+
+3. Fixes
+[core] Fix long job start times [#1806, #1825]
+[core] Fix CLI logging [#1820]
+[core] Make error messages more descriptive [#1951]
+[core] Fix deserialization exception for Object[] when using remote map sink [#1444]
+[core] Change join job operation to not be retried on failure [#1656]
+[core] Fix OperationTimeoutException when join request is sent to non-master node [#1661]
+[core] Fix race in job coordination service [#1680]
+[core] Fix multiple bugs in JDBC sink affecting jobs with timestamps [#1678, #1682]
+[core] Avoid nested wildcards in flatMap [#1736]
+[core] Fix job related methods misbehaving when master is in the process of going down [#1624]
+[core] Fix inconsistencies with @Nonnull and @Nullable annotations [#1675]
+[core] Fix "Buffer full" error when saving large snapshots [#1769]
+[core] Fix fragmentation type error in saving snapshots [#1771]
+[core] Fix merging of differently-timestamped streams [#1776]
+[core] Fix a problem in Kafka source, when used with no processing guaranties [#1705, #1831]
+[core] Get rid of metrics related warning messages [#1699]
+[core] Fix a data loss bug manifesting on batch job completion [#1861]
+[core] Fix a data loss bug affecting the S3 sink [#1629]
+[core] Fix a bug causing the emission old watermarks [#1879]
+[core] Remove checked exceptions from JobConfig [#1896]
+[jet-cli] Fix jet.sh submit to not report error messages as NULL [#1717]
+[integration] Fix RabbitMQ sink related issues [#1886]
+
```

Comment:
Shouldn't we document breaking changes here? We did in 3.1...

Reply:
it's there in the migration guide, but we should include here too.